[2025-06-10T20:32:29.313+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sentinel_ndvi_processing_pipeline.calculate_ndvi_from_bands manual__2025-06-10T20:32:23.241126+00:00 [queued]>
[2025-06-10T20:32:29.318+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sentinel_ndvi_processing_pipeline.calculate_ndvi_from_bands manual__2025-06-10T20:32:23.241126+00:00 [queued]>
[2025-06-10T20:32:29.318+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-06-10T20:32:29.324+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): calculate_ndvi_from_bands> on 2025-06-10 20:32:23.241126+00:00
[2025-06-10T20:32:29.327+0000] {standard_task_runner.py:60} INFO - Started process 63 to run task
[2025-06-10T20:32:29.329+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sentinel_ndvi_processing_pipeline', 'calculate_ndvi_from_bands', 'manual__2025-06-10T20:32:23.241126+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/process_sentinel_ndvi_dag.py', '--cfg-path', '/tmp/tmpedmmljgo']
[2025-06-10T20:32:29.330+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask calculate_ndvi_from_bands
[2025-06-10T20:32:29.357+0000] {task_command.py:423} INFO - Running <TaskInstance: sentinel_ndvi_processing_pipeline.calculate_ndvi_from_bands manual__2025-06-10T20:32:23.241126+00:00 [running]> on host 0afa38759197
[2025-06-10T20:32:29.399+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sentinel_ndvi_processing_pipeline' AIRFLOW_CTX_TASK_ID='calculate_ndvi_from_bands' AIRFLOW_CTX_EXECUTION_DATE='2025-06-10T20:32:23.241126+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-10T20:32:23.241126+00:00'
[2025-06-10T20:32:29.400+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-10T20:32:29.400+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/processing_output && python3 /opt/***/dags/pipeline_tasks/calculate_ndvi_from_bands.py --- Script Finished ---']
[2025-06-10T20:32:29.406+0000] {subprocess.py:86} INFO - Output:
[2025-06-10T20:32:29.945+0000] {subprocess.py:93} INFO - usage: calculate_ndvi_from_bands.py [-h] input_json
[2025-06-10T20:32:29.945+0000] {subprocess.py:93} INFO - calculate_ndvi_from_bands.py: error: unrecognized arguments: --- Finished ---
[2025-06-10T20:32:29.987+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-06-10T20:32:29.993+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-06-10T20:32:29.994+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=sentinel_ndvi_processing_pipeline, task_id=calculate_ndvi_from_bands, execution_date=20250610T203223, start_date=20250610T203229, end_date=20250610T203229
[2025-06-10T20:32:30.006+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 9 for task calculate_ndvi_from_bands (Bash command failed. The command returned a non-zero exit code 2.; 63)
[2025-06-10T20:32:30.022+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-06-10T20:32:30.035+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
